src/Parser/MistweaverMonk/Modules/Items/Velens.js

```diff
@@ -34,7 +34,25 @@ class Velens extends Module {
       return;
     }
 
-    this.healing += calculateEffectiveHealing(event, LEGENDARY_VELENS_HEALING_INCREASE);
+    // Rewriting the effectiveHealing function here, to remove the final step of removing the raw overheal from the effective healing of the trinket.
```

Comment:
I do believe the overhealing has bearing on the effectiveness of the 15% heal increase as I explained earlier. I'll try to explain my PoV with an example:
Normal spell heals for 1,000 raw
With Velen's this is 1,150 raw. So Velen's increased the heal by 150 raw healing, that healing is **on top** of the original 1,000 raw healing the spell normally does.
Then comes overhealing;
If the spell overheals for 50, then the spell will have healed for 1,100. At this point Velen's effectively contributes 100 healing.
If the spell overheals for 100, then the spell will have healed for 1,050. So velen's effectiveness is 50;
If the spell overheals for 150, then the spell will have healed for the original 1,000 and Velen's increase was completely wasted.
If the spell overheals for 200, then 50 healing of the original spell was already overhealing, and the increased healing from Velen's did nothing (0).

If you ignore raw healing and look at the actual healing done you may see the last 950 healing and think 15% of that was contributed by Velen's, while in fact it was 0.